docs/xdr/features/detect/rules_catalog.md

```diff
@@ -134,28 +137,47 @@ The Rule creation form has the following sections:
 - **General definition of the rule:**
   The rule name is mandatory during the creation, it will be used to name the corresponding raised alerts by default. You can add an optional description below.
```

Comment:
```suggestion
- The rule name is mandatory during the creation, it will be used to name the corresponding raised alerts by default. You can add an optional description below.
```